actor/src/main/resources/reference.conf

```diff
@@ -1145,11 +1145,13 @@ pekko {
         # other platforms, will default to 1).
         ndots = default
 
-        # The policy used to generate dns transaction ids. Options are thread-local-random or secure-random.
-        # Defaults to thread-local-random similar to Netty, secure-random produces FIPS compliant random numbers but
-        # could block looking for entropy (these are short integers so are easy to bruit-force, use thread-local-random
-        # unless you really require FIPS compliant random numbers).
-        id-generator-policy = thread-local-random
+        # The policy used to generate dns transaction ids. Options are thread-local-random, enhanced-double-hash-random
```

Comment:
Could you put the values in quotes? This makes them stand out a bit more. 'bruit' should 'brute' as in brute force.

Could you change 'initial seed, thread-local-random' to this (with a full stop instead of a comma)?
```
initial seed. 'thread-local-random'
```

Reply:
Done